### PR TITLE
raidboss: adjust callouts for BLU

### PR DESCRIPTION
**Pull request by xiashtra**
Prep for 6.45 BLU update.

### CODE REVIEW

File: ui/raidboss/data/03-hw/dungeon/the_vault61.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ const triggerSet: TriggerSet<Data> = {
       id: 'TheVault61 Altar Candle',
       regex: /Altar Candle/,
       beforeSeconds: 5,
-      condition: (data) => data.role !== 'dps',
+      condition: (data) => data.role !== 'dps' || data.job === 'BLU',
```
Comment (quisquous) [inline ui/raidboss/data/03-hw/dungeon/the_vault61.ts:50]:
```suggestion
```

Everybody gets tankbuster calls these days.

File: ui/raidboss/data/03-hw/dungeon/the_vault.ts [hunk 2]
```diff
@@ -47,7 +47,7 @@ const triggerSet: TriggerSet<Data> = {
       id: 'The Vault Altar Candle',
       regex: /Altar Candle/,
       beforeSeconds: 5,
-      condition: (data) => data.role !== 'dps',
+      condition: (data) => data.role !== 'dps' || data.job === 'BLU',
```
Comment (quisquous) [inline ui/raidboss/data/03-hw/dungeon/the_vault.ts:50]:
```suggestion
```

Everybody gets tankbuster calls these days.

File: ui/raidboss/data/05-shb/raid/e12s.ts [hunk 3]
```diff
@@ -1076,12 +1076,7 @@ const triggerSet: TriggerSet<Data> = {
           ko: '멀리 유도하기',
         },
         partyUnder: {
-          en: 'Get Under',
-          de: 'Unter ihn',
-          fr: 'En dessous',
-          ja: 'ボスと貼り付く',
-          cn: '去脚下',
-          ko: '보스 안쪽으로',
+          en: 'Party Under',
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/raid/e12s.ts:1079]: Maybe these calls should be like the Apo/Peri calls in p12s, where tanks get "Tank Out (Party In)" and party gets "Party In (Tanks Out)" and BLU could just get either one of those, honestly.

File: ui/raidboss/data/05-shb/trial/wol-ex.ts [hunk 4]
```diff
@@ -377,7 +377,7 @@ const triggerSet: TriggerSet<Data> = {
       // This is still 1 second before this cast goes off, giving ~7 seconds before LB is needed.
       delaySeconds: 4,
       alarmText: (data, _matches, output) => {
-        if (data.role === 'tank')
+        if (data.role === 'tank' || data.job === 'BLU')
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/trial/wol-ex.ts:380]: Mmmm, maybe? What if blu doesn't get LB? This is fine for now I guess, but maybe should say something else if this isn't the way to handle this.

Comment (xiashtra) [reply ui/raidboss/data/05-shb/trial/wol-ex.ts:380]: They need some type of call at least, and I was going for minimum changes.  Currently, BLUs substitute Diamondback where you would use a tank LB, so they should get those calls.

File: ui/raidboss/data/05-shb/raid/e12s.ts (hunk 3 above)
Comment (xiashtra) [reply ui/raidboss/data/05-shb/raid/e12s.ts:1079]: They probably should be, but I was going for minimum effort changes just to have something, and also make it less confusing for the tank BLUs.

Comment (quisquous) [reply ui/raidboss/data/05-shb/raid/e12s.ts:1079]: Would you mind just replacing with the calls from Apo/Peri? If this were an older older fight I'd say whatever, but this is something that is going to be done on blu very shortly.

File: ui/raidboss/data/03-hw/dungeon/the_lost_city_of_amdapor_hard.ts [hunk 5]
```diff
@@ -243,7 +243,7 @@ const triggerSet: TriggerSet<Data> = {
       id: 'LostCityHard Kuribu Cure IV',
       type: 'StartsUsing',
       netRegex: { id: '15DF', source: 'Kuribu', capture: false },
-      condition: (data) => data.role === 'tank',
+      condition: (data) => data.role === 'tank' || data.job === 'BLU',
```
[the remark this answers is not in the document]
Comment (xiashtra) [reply ui/raidboss/data/03-hw/dungeon/the_lost_city_of_amdapor_hard.ts:246]: They should, I'm not sure why this was marked as only tank previously.  I'll fix this up.

**Comment by quisquous** [comment]
Thank you!!

**Pull request merged**